Cargo.toml

```
@@ -3,6 +3,22 @@ name = "rmathlib"
 description = "A Rust Port of R's C Library of Special Functions"
 version = "1.0.0"
```

```suggestion
version = "1.0.0"
include = ["src/", "LICENSE", "README.md"]
```
To ensure C is not bundled